### PR TITLE
:sparkles: Add 'capsule apply' CLI command for applying a CapsuleSpec

### PR DESCRIPTION
**Pull request by MatiasFrank**

### CODE REVIEW

File: pkg/rbac/actions.go [hunk 1]
```diff
@@ -75,6 +75,7 @@ const (
 	ActionCapsuleAbortRollout = "capsule:abortrollout"
 	// Stop rollout
 	ActionCapsuleStopRollout = "capsule:stoprollout"
+	ActionCapsuleApplySpec   = "capsule:applyspec"
```
Comment (Munklinde96) [inline pkg/rbac/actions.go:78]: Guess we should put this under deploy, as it is no longer its own endpoint

Comment (Munklinde96) [reply pkg/rbac/actions.go:78]: And add it in default permissions

Comment (MatiasFrank) [reply pkg/rbac/actions.go:78]: Ah that one needs to be removed actually. I dropped the endpoint in favour of baking the functionality into Deploy

File: cmd/rig/cmd/capsule/deploy/deploy.go [hunk 2]
```diff
@@ -53,6 +56,144 @@ func parseEnvironmentSource(value string) (capsule.EnvironmentSource_Kind, strin
...
+		}
+		bytes, err := os.ReadFile(file)
+		if err != nil {
+			return nil, err
+		}
+		spec, err := v1.YAMLToCapsuleSpecProto(bytes)
+		if err != nil {
+			return nil, err
+		}
+		return []*capsule.Change{{
```
Comment (andersjohnsen) [inline cmd/rig/cmd/capsule/deploy/deploy.go:190]: It should be fine to combine with this more changes, no?

File: cmd/rig/cmd/capsule/deploy/deploy.go [hunk 3]
```diff
@@ -53,6 +56,144 @@ func parseEnvironmentSource(value string) (capsule.EnvironmentSource_Kind, strin
 }
 
 func (c *Cmd) deploy(ctx context.Context, cmd *cobra.Command, args []string) error {
+	if currentFingerprint != "" && currentRolloutID != 0 {
+		return errors.InvalidArgumentErrorf("cannot give both --fingerprint and --current-rollout")
```
Comment (andersjohnsen) [inline cmd/rig/cmd/capsule/deploy/deploy.go:60]: I believe we can control this in the flags thingy, marking two flags an mutualyl exclusive

**Pull request merged**